File: random.c
#include <stdio.h> 
#include <stdlib.h> 
#include<time.h> 

//prints random number between lower and upper bound
void GetRandom(int lower, int upper) 
{ 
    int i; 
    int num = (rand() % (upper - lower + 1)) + lower; 
    printf("%d \n", num); 
} 

int main() 
{ 
    int lower = 1, upper = 100000;
    srand(time(0)); 

    GetRandom(lower, upper); 
  
    return 0; 
} 

File: random.exs
# The following piece of code prompts a user to insert a lower limit number and a high limit number and then it prints a random number between the two limits.
# To run this program run elixir random.exs

print_random_number = fn() ->
    {low_limit, _} = IO.gets("Lower Limit:\n") |> Integer.parse
    {high_limit, _} = IO.gets("Higher Limit\n") |> Integer.parse
    Enum.random(low_limit..high_limit)
end

IO.puts("\nRandom Number:  #{print_random_number.()}")
